client/src/main/resources/templates/js/htmlreporter.js

```diff
@@ -16,36 +16,36 @@
 $(document).ready(function () {
 
   "use strict";
```

Comment:
@mach6 Since we are from Java background I felt using " feels better compare to use '. Another point main.js also we used ", so better we will follow the same standard in all our JS files.